azurerm/internal/services/network/express_route_circuit_peering_resource.go

```diff
@@ -108,6 +108,54 @@ func resourceArmExpressRouteCircuitPeering() *schema.Resource {
 				},
 			},
 
+			"ipv6_peering": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"microsoft_peering_config": {
```

Comment:
can we just shorten this to
```suggestion
						"microsoft_peering": {
```